### DISCUSSION TITLE
Why is Github Copilot encountering temporary issues?

### DISCUSSION THREAD

**Discussion by arnon001** [community/community#28157] [Copilot Conversations]
I'm writing code in JS,
and its not working for me...,
(i have copilot from student pack)

**Comment by davecheney** [comment]
Hi @arnon001. I'm sorry you are having issues. Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by davecheney** [reply]
@arnon001 it looks like you _really_ need to upgrade your copilot extension.
